api/bots_api/bot_lib.py

```diff
@@ -66,7 +72,7 @@ def __init__(self, client):
                           ' up the zuliprc file correctly.')
             sys.exit(1)
 
-    def send_message(self, message):
+    def external_send_message(self, message):
```

Comment:
Would it make sense to have a consistent name (`send_message` in both places, instead of `external_send_message` and `embedded_send_message`) so that clients can call the `BotHandler` without having to worry about whether it's an `ExternalBotHandler` or an `EmbeddedBotHandler`?